Graphs/Table/index.js

```diff
@@ -710,6 +713,8 @@ class Table extends AbstractGraph {
         }
 
         this.scroll ? this.updateTableStatus({ currentPage: this.currentPage, event: events.PAGING }) : this.updateData();
+        this.headerData = this.getHeaderData(this.getInitialSort());
+        this.updateData();
```

Comment:
potentially you could run updateData twice: see above. In any case tested this and the solution does not work